pipeline/defaults/defaults.go

```diff
@@ -31,6 +34,9 @@ func (Pipe) Description() string {
 // Run the pipe
 func (Pipe) Run(ctx *context.Context) error {
 	ctx.Config.Dist = "dist"
```

Comment:
Perhaps it would make sense to factor all the name stuff into a function like the others, so maybe `setNameDefaults(ctx)`, etc?

Reply:
yeah, that makes sense...